app/assets/stylesheets/therapists/_show.scss

```diff
@@ -0,0 +1,16 @@
+.show-container {
+  display: flex;
+  padding: 40px 20px 20px 10px;
+  justify-content: space-around;
+}
+
+.show-first-flex-item div {
```

Comment:
will you please nest your classes as in SCSS style aka ;

.show-therapist-container {
.show-first-flex-item {
}
}